test/apps/gups.c

```diff
@@ -322,10 +322,10 @@ SHMEMRandomAccess(void)
   int64_t ProcNumUpdates; /* number of updates per processor */
```

Comment:
Oh, of course.  :man_facepalming:

Actually, I'm surprised the compiler didn't warn about loss of information passing int64_t into a uint64_t function argument..